MapboxSearch/place-autocomplete/src/main/java/com/mapbox/search/autocomplete/PlaceAutocompleteSuggestion.kt

```diff
@@ -27,7 +27,7 @@ public class PlaceAutocompleteSuggestion internal constructor(
     /**
      * Place geographic point.
      */
-    public val coordinate: Point,
+    public val coordinate: Point?,
```

Comment:
@eugenes78 this's an API breaking change. You'll need to update public api files https://github.com/mapbox/mapbox-search-android#public-api-changes-tracking

Also, this change should be described in the https://github.com/mapbox/mapbox-search-android/blob/main/CHANGELOG.md

Reply:
This field was removed completely.

Reply:
Updated  [CHANGELOG.md](CHANGELOG.md) accordingly